audit/projects/k8s-staging-sig-storage/iam.json

```diff
@@ -46,18 +46,12 @@
     },
     {
       "members": [
+        "serviceAccount:272675062337-compute@developer.gserviceaccount.com",
         "serviceAccount:272675062337@cloudservices.gserviceaccount.com",
-        "serviceAccount:service-272675062337@containerregistry.iam.gserviceaccount.com",
-        "serviceAccount:272675062337-compute@developer.gserviceaccount.com"
+        "serviceAccount:service-272675062337@containerregistry.iam.gserviceaccount.com"
```

Comment:
I'm mildly surprised these weren't alpha-sorted before.

It looks like the compute service account binding sticks around even after the compute service is disabled.

I'll bucket this under another thing to check out as part of https://github.com/kubernetes/k8s.io/issues/1665